src/SingleArticle.jsx

```diff
@@ -1,22 +1,43 @@
 import { useState, useEffect } from "react";
+import { useParams } from 'react-router-dom';
 import { getSingleArticle } from "./utils/api";
+import { format, parseISO } from 'date-fns'
+import './SingleArticle.css'
 
-const ArticleList = () => {
-
+const SingleArticle = () => {
+const {article_id} = useParams();
+const [formattedDate, setFormattedDate] = useState('')
 const [article, setSingleArticle] = useState({})
 useEffect(() => {
-    getSingleArticle()
+    getSingleArticle(article_id)
     .then((article) => {
-        setArticles(article)
+        setFormattedDate(format(new Date(`${article.created_at}`), 'EEE d MMM yyyy HH:mm a'))
+        setSingleArticle(article)
```

Comment:
It would be nice to see you implement a loading state for visual feedback for the user 👀